getAlby/lightning-browser-extension
# feat(toasts): add react toastify to test handing error popups

#### Link this PR to an issue
Fixes #812 

#### Type of change (Remove other not matching type)

- New feature (non-breaking change which adds functionality)


#### Describe the changes you have made in this PR -
added react-toastify to handle error messages not fitting in pop-up window.

#### Screenshots of the changes (If any) -

<img width="512" alt="Screen Shot 2022-05-11 at 18 24 25" src="https://user-images.githubusercontent.com/104893211/167828854-a0ecef8b-b164-48d6-84d5-9cf43df320e5.png">
<img width="1312" alt="Screen Shot 2022-05-11 at 18 25 02" src="https://user-images.githubusercontent.com/104893211/167828874-b7f5a60f-d26f-4d14-8787-e0753b81e498.png">
<img width="1312" alt="Screen Shot 2022-05-11 at 18 25 16" src="https://user-images.githubusercontent.com/104893211/167828881-8cc4bbc8-a428-4e2e-a21e-a6de4e27f268.png">
<img width="1312" alt="Screen Shot 2022-05-11 at 18 25 36" src="https://user-images.githubusercontent.com/104893211/167828887-d640226d-1761-4baa-b5cc-bd9391cfe9c7.png">
<img width="1312" alt="Screen Shot 2022-05-11 at 18 31 07" src="https://user-images.githubusercontent.com/104893211/167829690-35e1b882-ba61-4588-9ace-bd02c8adacaa.png">


#### How Has This Been Tested?

Case 1
      1. go to getalby.com/demo
      2. Try to make a payment with a balance that is lower then 25 sats (+fees)
      3. See error
Case 2 
      1. click on popup, go to send
      2. enter invalid lnurl or invoice or lrurl address
      3. see error

#### Checklist:

- [x] My code follows the style guidelines of this project and performed a self-review of my own code
- [x] I checked if I need to make corresponding changes to the documentation (and made those changes if needed)


fixes #919
